### PR TITLE
feat: Canteen -  Frontend

### PR DESCRIPTION
**Pull request by Natalija9**
Added:
- Show user meals (Implements #138)
- Enable ordering new meals and show basket (Implements #153)
- Show confirmation dialog on checkout (Implements #154)
- Add successful transaction notification (Implements #155)

Additional information:
- Temporary disabled protection on Canteen service (needed for testing)

### CODE REVIEW

File: DormService/DormApps/dorm-spa/src/Apps/CanteenPage/CanteenPage.tsx [hunk 1]
```diff
@@ -1,7 +1,185 @@
...
+      console.log(response);
+
+      setUserMealsData(response);
+      setShowNotification({type: NotificationType.Success, message: "Transaction completed successfully!"}); // OK
+    })
+    .catch(() => setShowNotification({type: NotificationType.Error, message: "Something went wrong, check you credit status!"})); // Something went wrong
+
+  }
+
+  const chechoutOrder = () => {
```
Comment (momciloknezevic7) [inline DormService/DormApps/dorm-spa/src/Apps/CanteenPage/CanteenPage.tsx:49]: Typo fix.

File: DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs [hunk 2]
```diff
@@ -74,14 +86,15 @@ public async Task<ActionResult> DeleteOrder(string username)
...
-            }
+            //if (User.FindFirst(ClaimTypes.Name).Value != username)
+            //{
+            //    return Forbid();
+            //}
 
             var order = await _repository.GetOrder(username);
             if (order == null)
             {
+                _logger.LogInformation("No order----------------------------------");
```
Comment (momciloknezevic7) [inline DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs:97]: Remove log line.

**Review by momciloknezevic7 — APPROVED** [review]

File: DormService/DormApps/dorm-spa/src/Apps/CanteenPage/CanteenPage.tsx (hunk 1 above)
Comment (Natalija9) [reply DormService/DormApps/dorm-spa/src/Apps/CanteenPage/CanteenPage.tsx]: Nice catch, fixed it.

File: DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs (hunk 2 above)
Comment (Natalija9) [reply DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs]: Done.

File: DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs [hunk 3]
```diff
@@ -122,14 +135,16 @@ public async Task<ActionResult<UserMeals>> Checkout(string username)
...
                     return Accepted();
                 }
             }
             catch (RpcException e)
             {
                 _logger.LogInformation("Error while calling Payment service: {message}",  e.Message);
-            }           
+            }
+
+            _logger.LogInformation("End----------------------------------");
```
[the remark this answers is not in the document]
Comment (Natalija9) [reply DormService/Services/Canteen/Canteen.API/OrderMealsInfo/Controllers/OrderMealsController.cs]: Done.

**Pull request merged**